spec/koans.js

```diff
@@ -457,27 +467,27 @@ describe('assign object property values to new variables while destructuring. ',
 
   describe('for simple objects', function() {
     it('use a colon after the property name, like so `propertyName: newName`', () => {
-      const {x} = {x: 1};
-      //expect(y).toEqual(1);
+      const {x: y} = {x: 1};
+      expect(y).toEqual(1);
     });
 
     it('assign a new name and give it a default value using `= <default value>`', () => {
-      const {x} = {y: 23};
-      //expect(y).toEqual(42);
+      const {x: y=42} = {y: 23};
+      expect(y).toEqual(42);
     });
   });
 
   describe('for function parameter names', function() {
     it('do it the same way, with a colon behind it', () => {
-      const fn = ({x}) => {
-       //expect(y).toEqual(1);
+      const fn = ({x, y}) => {
+       expect(y).toEqual(1);
       };
-      fn({x: 1});
+      fn({x: 1, y:1});
```

Comment:
This test is about assigning a new name to a function parameter, so if you do : 
```js
      const fn = ({x:y}) => {
       expect(y).toEqual(1);
      };
```
When you call the function with a value of 1 for x:
  fn({x: 1});
then y is also equal to 1!

Reply:
```js
describe('for function parameter names', function() {
    it('do it the same way, with a colon behind it', () => {
      const fn = ({x:y}) => {
       expect(y).toEqual(1);
      };
      fn({x: 1});
    });
```

Reply:
👍